src/test/unit/engine/NetworkTest.cpp

```diff
@@ -24,542 +24,542 @@
  * Implementation of Network test
  */
 
-#include "NetworkTest.hpp"
+#include "gtest/gtest.h"
 
 #include <nupic/engine/Network.hpp>
 #include <nupic/engine/NuPIC.hpp>
 #include <nupic/engine/Region.hpp>
 #include <nupic/ntypes/Dimensions.hpp>
+#include <nupic/utils/Log.hpp>
 
 using namespace nupic;
 
+#define SHOULDFAIL_WITH_MESSAGE(statement, message) \
```

Comment:
I think there are some included gtest macros you can use that are explained here:
https://github.com/google/googletest/blob/master/googletest/docs/AdvancedGuide.md#including-this-header-you-can-use